local-proxy/src/server.ts

```diff
@@ -66,10 +66,20 @@ app.post('/invoke', json(), async (req, res) => {
   const { path, handler, args } = req.body;
   registry.register(requestId);
   try {
-    await transpilePromise;
-    // Make sure we use tmpDir as absolute path
-    const mod = require(pathJoin(tmpDir, path));
-    const fn = mod[handler];
+    let fn: (...args: any[]) => any;
+    if (path === '@binaris/shift-db' && (handler === 'getVersioned' || handler === 'poll')) {
+      fn = require(path)[handler];
```

Comment:
This guarantees that `local-proxy` depends on `db`, why not put it in dependencies and import it on start?

Reply:
Sounds good